### DISCUSSION TITLE
Flashlight Dim Triggers Earlier Than Expected

### DISCUSSION THREAD

**Discussion by SevenEnd7** [ppy/osu#26377] [Q&A] [answered]
Currently, flashlight dim triggers if you're hitting notes on top of a slider head (clicking w/o notes triggers this too). The original interaction only triggers when you're actively holding onto a slider while the sliderball is active. 
[video example (lazer)](https://github.com/ppy/osu/assets/69742831/732d7364-a7e0-49d1-81f5-3b44f06e0734)
[video example (stable)](https://github.com/ppy/osu/assets/69742831/0afd7658-a553-496e-a24f-6703c825e29d)

[map used](https://osu.ppy.sh/beatmapsets/2111522#osu/4439185)

The new slider leniency change is what likely caused this new interaction ([slider leniency change here](https://github.com/ppy/osu/commit/599fdb0128d88cbc44b8147cd5058b83528f2287)). Someone else smarter than me could find out the specific reason why this happens on a technical level. 

[compressed-logs.zip](https://github.com/ppy/osu/files/13826119/compressed-logs.zip)

**Comment by Taoshix** [comment]
I was also just about to report this, but then i noticed this thread.
This happened to me as well.

https://github.com/ppy/osu/assets/9339576/f5085226-7e64-41b2-98b3-f9befc2c6469

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/26515
